ci-operator/config/openshift/release/openshift-release-master__nightly-4.11.yaml

```diff
@@ -118,15 +118,7 @@ tests:
         IPv4=no
         VIP_DHCP_ALLOCATION=no
         PROXY=yes
-    workflow: baremetalds-assisted
-- as: e2e-aws-single-node-workers
-  interval: 24h
-  steps:
-    cluster_profile: aws-2
-    env:
-      BASE_DOMAIN: aws-2.ci.openshift.org
-    workflow: openshift-e2e-aws-single-node-workers
-  timeout: 3h0m0s
+    workflow: assisted-baremetal
```

Comment:
why it's gone?

Reply:
I didn't deleted it manually but i think it renamed to "openshift-e2e-aws-single-node-workers"